src/header-generator.js

```diff
@@ -0,0 +1,284 @@
+const fs = require('fs');
+const path = require('path');
+const { BayesianNetwork } = require('bayesian-network');
+const { default: ow } = require('ow');
+
+const headerNetworkDefinitionPath = path.join(__dirname, './data_files/header-network-definition.json');
+const inputNetworkDefinitionPath = path.join(__dirname, './data_files/input-network-definition.json');
+const browserHelperFilePath = path.join(__dirname, './data_files/browser-helper-file.json');
```

Comment:
What exactly should this file be? The pre-trained models?

Reply:
Yeah. There will be two more repositories with pull requests in the next few days, first one is the bayesian network, second is the one that creates these files from data.